### ISSUE TITLE
Update internship application status

### ISSUE THREAD

**Issue by kbwagner** [openopps/OpenOppsTasks#199]
Update applicant status help content for internships based on the following: 

Not complete - when a student is a primary selection and the Hiring Manger does not mark them complete before closing the internship.
Completed - when a student is an alternate or primary selection and the Hiring Manger marks them as completed before closing the internship.
Alternate - when a student is an alternate selection and the Hiring Manger does not mark them completed before closing the internship

**Comment by dalenard21** [comment]
@kbwagner 

So, I had updated this page with the last release and I thought we had decided on "Assigned" and "Not Assigned", but looks like you all decided on just  "Alternate"?  Not a big deal, but I recall there was much back and forth and thought we had made a decision.  

I'll update the page to remove "Assigned" and "Not Assigned" and put in Alternate.  Can you please review the other descriptions on the page to make sure they are accurate?  Here's the page in production:

https://usajobs.github.io/openopps-help/opportunity/application-status/#application-status-for-student-internship-opportunities

I'm not going to update the descriptions to say "when the "xyz" email is sent from ATP...", because the student applicants will have no idea what that means.  We need the descriptions to tell the students what the status means to them, not internally for us.

Thanks.

**Comment by kbwagner** [comment]
@dalenard21 @Piglady64 julie and I chatted about this on Friday and came up with the labels so a bit different than proposed.  But if you have strong feelings either way just let us know.

The descriptions I gave you are the ones we gave Macon not want I thought you should use.

Will review in just a bit.

**Comment by kbwagner** [comment]
@dalenard21 yes it's all accurate minus the ones we changed.  sorry for the change...as we talked through the tickets and values more it just seemed like we needed to call our Alternate.

**Comment by dalenard21** [comment]
@kbwagner no problem.  Just double checking.  I'll make the update this week.  thanks!

**Comment by dalenard21** [comment]
@kbwagner here are a couple of options to describe the 'alternate' status.

1. Alternate
You are an alternate for the internship. You may be selected for the internship if the primary candidate decides not to participate. 

2. You've been selected as an alternate for the internship. The internship coordinator has chosen a primary candidate for the internship.  If the primary candidate does not accept their offer,  you may move into the primary position. 

Thoughts?

**Comment by kbwagner** [comment]
@dalenard21 I think I like #1 better.  But...it's actually more than the primary not deciding to accept. They might not pass a background check is the bigger reason I think.  So maybe something like if the primary candidate does not accept or is unable to move forward...?

**Comment by dalenard21** [comment]
@kbwagner Do you think we need to explain it could be a background issue?  Maybe we just say the primary candidate fell through. 

You are an alternate for the internship. You may be selected for the internship if the primary candidate falls through.

or, very detailed:

You are an alternate for the internship.  You may be selected for the internship if the primary candidate does not accept their offer, or does not pass the background check.

**Comment by kbwagner** [comment]
@dalenard21 I think the generic version is best since there could be any number of reasons they don't get it or show up.

**Comment by dalenard21** [comment]
@kbwagner agree.  I'll make the update. thanks for your review.

**Comment by dalenard21** [comment]
@kbwagner I updated the page to remove "Assigned" and "Not Assigned" as statuses for the student internship applications, and added a new status of "Alternate".  Please take another look to make sure it all is good.

https://federalist-proxy.app.cloud.gov/site/usajobs/openopps-help/opportunity/application-status/

Then we can get this pushed out with the next release.

**Comment by kbwagner** [comment]
@dalenard21 - Submitted is actually applied.  Can you update that when you return?  My fault!

**Comment by dalenard21** [comment]
@kbwagner Changed "Submitted" to "Applied"
https://federalist-proxy.app.cloud.gov/site/usajobs/openopps-help/opportunity/application-status/

**Comment by kbwagner** [comment]
@bmcorum we have an update to some help content.  We usually push out with the release (but I wasn't here to nag).  Let me know when we can get this pushed out.  Or, it can probably wait til 8.0 if you prefer?

**Comment by dalenard21** [comment]
@bmcorum @kbwagner yep, content is ready to go whenever you can do the push.  thanks.
